core/src/main/scala/org/apache/spark/util/logging/RollingFileAppender.scala

```diff
@@ -97,11 +125,11 @@ private[spark] class RollingFileAppender(
           altRolloverFile = new File(activeFile.getParent,
             s"${activeFile.getName}$rolloverSuffix--$i").getAbsoluteFile
```

Comment:
Want to double check. If we enable compression, the suffix will still be gz even we use the alternative file name, right?